src/MongoDB.Driver.Core/MongoDB.Driver.Core.csproj

```diff
@@ -77,12 +77,35 @@
     <Compile Include="..\MongoDB.Shared\MaxTimeHelper.cs" Link="Core\Operations\MaxTimeHelper.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
```

Comment:
I don't understand these `.csproj` file changes at all so I'm deferring to the you and Dmitry.